peachjam/admin.py

```diff
@@ -34,14 +34,20 @@
     Legislation,
     Locality,
     MatterType,
+    PeachJamSettings,
     Predicate,
     Relationship,
     SourceFile,
     Taxonomy,
+    pj_settings,
 )
 from peachjam.resources import GenericDocumentResource, JudgmentResource
 
 
+class PeachJamSettingsAdmin(admin.ModelAdmin):
+    filter_horizontal = ("document_languages",)
```

Comment:
```suggestion
    filter_horizontal = ("document_languages", "document_jurisdictions",)
```
We can use this for both languages and jurisdictions.